tests/js/spec/components/indicators.spec.jsx

```diff
@@ -12,184 +18,164 @@ import IndicatorStore from 'app/stores/indicatorStore';
 // Make sure we use `duration: null` to test add/remove
 jest.useFakeTimers();
 
+jest.mock('framer-motion', () => ({
+  ...jest.requireActual('framer-motion'),
+  AnimatePresence: jest.fn(({children}) => children),
+}));
+
+function createWrapper(props = {}) {
+  const {routerContext} = initializeOrg(props);
+  return mountWithTheme(<Indicators />, {context: routerContext});
+}
+
 describe('Indicators', function () {
   let wrapper;
```

Comment:
if you want autocomplete next time https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html#type
```suggestion
  /** @type {ReturnType<typeof createWrapper>} */
  let wrapper;
```